platforms/core-configuration/declarative-dsl-evaluator/src/main/kotlin/org/gradle/internal/declarativedsl/evaluator/defaults/ModelDefaultsBlockSchema.kt

```diff
@@ -27,6 +29,9 @@ interface DefaultsTopLevelReceiver {
     @Configuring
     @AccessFromCurrentReceiverOnly
     fun defaults(defaults: DefaultsConfiguringBlock.() -> Unit)
+
+    @get:Restricted
+    val layout: ProjectLayout
```

Comment:
❓ Shouldn't this be in `DefaultsConfiguringBlock` instead?

Reply:
Another potential improvement, which we might make in the future, is placing it in some interface like `HasLayouts` (`HasProjectLayout`, `HasBuildLayout`?)